static/src/js/components/SearchForm/SearchForm.js

```diff
@@ -123,6 +137,20 @@ class SearchForm extends Component {
     onToggleAdvancedSearchModal(true)
   }
 
+  onWindowKeyDown(e) {
+    const { key } = e
+    const { inputRef, keyboardShortcuts } = this
+    const inputIsRendered = inputRef.current && inputRef.current.input
+
+    // Focus the search input when the forward slash key is pressed
+    if (key === keyboardShortcuts.focusSearchInput && inputIsRendered) {
+      inputRef.current.input.focus()
+
```

Comment:
I think we can have a block here that checks to see if the input is currently already focused, and if so, allows the `/` to be entered in the input.

Reply:
Makes sense!